src/controller-types/src/dyncfgs.rs

```diff
@@ -32,6 +32,18 @@ pub const WALLCLOCK_LAG_HISTORY_REFRESH_INTERVAL: Config<Duration> = Config::new
     "The interval at which to refresh `WallclockLagHistory` introspection.",
 );
 
+pub const WALLCLOCK_LAG_HISTOGRAM_REFRESH_INTERVAL: Config<Duration> = Config::new(
+    "wallclock_lag_histogram_refresh_interval",
+    Duration::from_secs(60),
+    "The interval at which to refresh `WallclockLagHistogram` introspection.",
+);
+
+pub const WALLCLOCK_LAG_HISTOGRAM_PERIOD_LENGTH: Config<Duration> = Config::new(
+    "wallclock_lag_histogram_period_length",
+    Duration::from_secs(24 * 60 * 60),
+    "The period length of histograms in `WallclockLagHistogram` introspection.",
+);
+
```

Comment:
Maybe interval would be more specific than length?

Reply:
SGTM!